app/components/npq_separation/admin/statement_details_component.html.erb

```diff
@@ -0,0 +1,108 @@
+<div class="app-application__panel__summary statement-details-component">
+  <div class="govuk-!-margin-right-4">
+    <h4 class="govuk-heading-l govuk-!-margin-bottom-2"><%= t("statements.totals.heading") %></h4>
+
+    <p class="govuk-body-s govuk-!-margin-bottom-2">
+      <%= t("statements.totals.output_payment") %>
+      <span><%= helpers.number_to_pounds(@calculator.total_output_payment) %></span>
+    </p>
+
+    <% if @calculator.show_targeted_delivery_funding? %>
+      <p class="govuk-body-s govuk-!-margin-bottom-2">
+        <%= t("statements.totals.targeted_delivery_funding") %>
+        <span><%= helpers.number_to_pounds(@calculator.total_targeted_delivery_funding) %></span>
+      </p>
+    <% end %>
+
+    <p class="govuk-body-s govuk-!-margin-bottom-2">
+      <%= t("statements.totals.clawbacks") %>
+      <span><%= helpers.number_to_pounds(-@calculator.total_clawbacks) %></span>
+    </p>
+
+    <% unless @calculator.total_service_fees.zero? %>
+      <p class="govuk-body-s govuk-!-margin-bottom-2">
+        <%= t("statements.totals.service_fee") %>
+        <span><%= helpers.number_to_pounds(@calculator.total_service_fees) %></span>
+      </p>
+    <% end %>
+
+    <hr/>
+
+    <p class="govuk-body-s govuk-!-margin-bottom-2">
+      <strong><%= t("statements.totals.total_net_vat") %></strong>
+      <strong><span><%= helpers.number_to_pounds(@calculator.total_payment) %></span></strong>
+    </p>
+  </div>
+
+  <ul class="first govuk-list govuk-!-margin-bottom-0">
+    <li>
+      <strong class="float govuk-body-s govuk-!-margin-bottom-0"><%= t("statements.cut_off_date") %></strong>
+      <div class="number-group">
+        <div class="number-container govuk-!-padding-right-4">
+          <div class="float tooltip">
+            <div class="float govuk-heading-m govuk-!-margin-bottom-0 govuk-!-padding-top-0">
+              <span><%= @statement.deadline_date.to_fs(:govuk) %></span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </li>
+  </ul>
+  <ul class="second govuk-list govuk-!-margin-bottom-0">
+    <li>
+      <strong class="float govuk-body-s govuk-!-margin-bottom-0"><%= t("statements.total_starts") %></strong>
+      <div class="number-group">
+        <div class="number-container govuk-!-padding-right-4">
+          <div class="float tooltip">
+            <div class="float govuk-heading-m govuk-!-margin-bottom-0 govuk-!-padding-top-0">
+              <span><%= @calculator.total_starts %></span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </li>
+  </ul>
+  <ul class="booster govuk-list govuk-!-margin-bottom-0">
+    <li>
+      <strong class="govuk-body-s float govuk-!-margin-bottom-0"><%= t("statements.total_retained") %></strong>
```

Comment:
This would all benefit from a rewrite to use definition lists but for expediency I went with what was already in place on ECF